Cargo.lock

```diff
@@ -1,21 +0,0 @@
-[root]
```

Comment:
This is my personal habit, because I think `cargo` will generate this file by itself, so there is no need to put it in the source code for tracking.
I respect different habits, so I will submit this file later.